pkg/chain/ethereum/tbtc.go

```diff
@@ -466,6 +468,18 @@ func (ta *tbtcApplication) getDepositContract(
 	return depositContract, nil
 }
 
+func parseUtxoOutpoint(utxoOutpoint []uint8) (string, uint32) {
```

Comment:
We're duplicating the code and tests in `celo` and `ethereum` packages. We could extract it to some common place. I'm not sure where though. Maybe `pkg/chain/tbtc.go`, or other file under `pkg/chain`?